google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GrpcSpannerRpc.java

```diff
@@ -480,6 +490,37 @@ public void start(Listener<RespT> responseListener, Metadata metadata) {
     return elements.get((int) index);
   }
 
+  /**
+   * This is a one time setup for grpcz pages. This adds all of the methods to the Tracing
+   * environment required to show a consistent set of methods relating to Cloud Bigtable on the
+   * grpcz page.  If HBase artifacts are present, this will add tracing metadata for HBase methods.
+   *
+   * TODO: Remove this when we depend on gRPC 1.8
+   */
+  private static void setupTracingConfig() {
```

Comment:
Should this be configured in Rpc class (looks like a job for the utils class)?

Reply:
This will be removed pretty soon since grpc will do it automatically with 1.8. So I did not bother to put it in a separate class.

Reply:
This is the error I get
```
[WARNING] Rule 3: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for com.google.errorprone:error_prone_annotations:2.0.19 paths to dependency are:
+-com.google.cloud:google-cloud-spanner:0.32.1-beta-SNAPSHOT
  +-com.google.truth:truth:0.30
    +-com.google.errorprone:error_prone_annotations:2.0.19 (managed) <-- com.google.errorprone:error_prone_annotations:2.0.8
and
+-com.google.cloud:google-cloud-spanner:0.32.1-beta-SNAPSHOT
  +-com.google.guava:guava-testlib:20.0
    +-com.google.errorprone:error_prone_annotations:2.0.19 (managed) <-- com.google.errorprone:error_prone_annotations:2.0.12
and
+-com.google.cloud:google-cloud-spanner:0.32.1-beta-SNAPSHOT
  +-io.opencensus:opencensus-api:0.9.0 (managed) <-- io.opencensus:opencensus-api:0.6.0
    +-com.google.errorprone:error_prone_annotations:2.0.19
and
+-com.google.cloud:google-cloud-spanner:0.32.1-beta-SNAPSHOT
  +-io.opencensus:opencensus-contrib-grpc-util:0.9.0
    +-com.google.errorprone:error_prone_annotations:2.0.19 (managed) <-- com.google.errorprone:error_prone_annotations:2.1.2
and
+-com.google.cloud:google-cloud-spanner:0.32.1-beta-SNAPSHOT
  +-io.grpc:grpc-netty:1.7.0
    +-io.grpc:grpc-core:1.7.0
      +-com.google.errorprone:error_prone_annotations:2.0.19 (managed) <-- com.google.errorprone:error_prone_annotations:2.1.2
]
```